projects/1.19/assets/atmospheric/atmospheric/lang/zh_cn.json

```
@@ -0,0 +1,547 @@
+  "advancements.atmospheric.end.forbidden_fruit.description": "你的食物逃跑了...",
+  "advancements.atmospheric.end.forbidden_fruit.title": "禁忌之果",
+  "advancements.atmospheric.husbandry.find_orange.description": "找到长在月桂树上的橙子",
+  "advancements.atmospheric.husbandry.find_orange.title": "有橙子了，你高兴吗？",
+  "advancements.atmospheric.husbandry.finish_gateau.description": "吃下最后一片丝兰果大蛋糕",
+  "advancements.atmospheric.husbandry.finish_gateau.title": "何不食...蛋糕？",
+  "advancements.atmospheric.husbandry.kill_mob_with_passion_fruit.description": "用百香果的种子射杀一个生物",
+  "advancements.atmospheric.husbandry.kill_mob_with_passion_fruit.title": "激情犯罪",
+  "advancements.atmospheric.husbandry.obtain_blood_orange.description": "通过在下界种植月桂树以获得血橙",
+  "advancements.atmospheric.husbandry.obtain_blood_orange.title": "鲁多维科科技",
+  "advancements.atmospheric.husbandry.persistence_while_starving.description": "在最饥饿的时候获得持久效果",
+  "advancements.atmospheric.husbandry.persistence_while_starving.title": "快餐快人",
+  "advancements.atmospheric.husbandry.put_out_fire.description": "在着火时用芦荟叶自救",
+  "advancements.atmospheric.husbandry.put_out_fire.title": "燃烧吧，宝贝！",
+  "advancements.atmospheric.husbandry.spit_passion_fruit.description": "用吃百香果后吐出的百香果种子射击生物",
+  "advancements.atmospheric.husbandry.spit_passion_fruit.title": "噗——",
+  "biome.atmospheric.aspen_parkland": "白杨林地",
+  "biome.atmospheric.dunes": "沙丘",
+  "biome.atmospheric.flourishing_dunes": "繁茂沙丘",
+  "biome.atmospheric.grimwoods": "阴郁树林",
+  "biome.atmospheric.hot_springs": "热泉",
+  "biome.atmospheric.kousa_jungle": "梾树丛林",
+  "biome.atmospheric.laurel_forest": "月桂森林",
+  "biome.atmospheric.petrified_dunes": "石化沙丘",
+  "biome.atmospheric.rainforest": "雨林",
+  "biome.atmospheric.rainforest_basin": "雨林盆地",
+  "biome.atmospheric.rocky_dunes": "多岩沙丘",
+  "biome.atmospheric.scrubland": "灌丛地",
+  "biome.atmospheric.snowy_scrubland": "积雪灌丛地",
+  "biome.atmospheric.sparse_rainforest": "稀疏雨林",
+  "biome.atmospheric.sparse_rainforest_basin": "稀疏雨林盆地",
+  "biome.atmospheric.spiny_thicket": "多刺树丛",
+  "block.atmospheric.agave": "龙舌兰",
+  "block.atmospheric.aloe_bundle": "芦荟捆",
+  "block.atmospheric.aloe_gel_block": "芦荟凝胶块",
+  "block.atmospheric.aloe_vera": "芦荟",
+  "block.atmospheric.arid_sand": "干燥沙子",
+  "block.atmospheric.arid_sandstone": "干燥砂岩",
+  "block.atmospheric.arid_sandstone_brick_slab": "干燥砂岩砖台阶",
+  "block.atmospheric.arid_sandstone_brick_stairs": "干燥砂岩砖楼梯",
+  "block.atmospheric.arid_sandstone_brick_vertical_slab": "干燥砂岩砖竖直台阶",
+  "block.atmospheric.arid_sandstone_brick_wall": "干燥砂岩砖墙",
+  "block.atmospheric.arid_sandstone_bricks": "干燥砂岩砖",
+  "block.atmospheric.arid_sandstone_slab": "干燥砂岩台阶",
+  "block.atmospheric.arid_sandstone_stairs": "干燥砂岩楼梯",
+  "block.atmospheric.arid_sandstone_vertical_slab": "干燥砂岩竖直台阶",
+  "block.atmospheric.arid_sandstone_wall": "干燥砂岩墙",
+  "block.atmospheric.arid_sprouts": "旱地苗",
+  "block.atmospheric.aspen_beehive": "白杨木蜂箱",
+  "block.atmospheric.aspen_boards": "白杨竖纹板",

```

```suggestion
  "block.atmospheric.aspen_boards": "白杨木直纹板",
```
参照库内Woodworks翻译，请全文通常